pkg/services/rendering/http_mode.go

```diff
@@ -180,6 +182,10 @@ func (rs *RenderingService) doRequest(ctx context.Context, u *url.URL, headers m
 		return nil, fmt.Errorf("failed to send request to remote rendering service: %w", err)
 	}
 
+	if resp.StatusCode == http.StatusTooManyRequests {
```

Comment:
I imagine we can configure what too many requests looks like via renderer config in deployment tools repo?

Reply:
Yes exactly.